### PR TITLE
Fix clear_screen on Windows

### PR DESCRIPTION
**Pull request by gwenn**
We should both clear content and attributes.
See #366

### CODE REVIEW

File: src/tty/windows.rs [hunk 1]
```diff
@@ -296,6 +296,13 @@ impl ConsoleRenderer {
             pos,
             &mut _count,
         ));
+        check!(wincon::FillConsoleOutputAttribute(
+            self.handle,
+            attr,
+            length,
+            pos,
+            &mut _count
```
Comment (bjorn3) [inline src/tty/windows.rs:304]:
```suggestion
            &mut _count,
```

Comment (gwenn) [reply src/tty/windows.rs:304]: `cargo fmt` doesn't add a comma !

Comment (bjorn3) [reply src/tty/windows.rs:304]: That is because this is inside a macro, so it doesn't know it is allowed to do that.

Comment (gwenn) [reply src/tty/windows.rs]: Indeed !

**Pull request merged**